manuscript/manuscript.Rmd

```diff
@@ -112,11 +110,35 @@ Rich, processes-based simulations such as the SORTIE model in forest management
 Decision-theoretic approaches based on optimal control techniques can only find the best strategy in the simplest of ecological models; the so called "curse of dimensionality" makes problems with a large number of states or actions intractable by conventional methods [@Possingham2006; @Marescot2013; @ferrer-mestres_k-n-momdps_2021; @chades_primer_2021].
```

Comment:
Maybe assuming previous knowledge of "Decision-theoretic approaches" is appropriate for the intended audience, but might be helpful to have one more sentence of description of the method and/or how it relates to RL. Or just a couple words outlining that this is current best practice approach?